### ISSUE TITLE
Better info (especially flashing)

### ISSUE THREAD

**Issue by Martinsos** [LifeIsOnTheWire/Postage-Board#2]
Hi!

I love the work you have done with the Postage Board, it is really great for building handwired keyboards!

I just used it for my first hand-wire ever and although documentation is pretty good, I wanted to share some parts where I got stuck or where I found that information is missing, so that you can potentially improve it for the future users.

1. Documentation is dissipated.  There is some info on GeekHack, some on reddit and some here on github. I bought it on keeb.io and having one link to detailed specifications and instructions was what I expected, but I could not find. I think it would be great merging all those docs together here in Github. I will numerate specific things below.
2. Add example photo -> there is nice example photo on geekhack, it would be great if you could put it here, it helped me a lot. Although what I don't understand on that photo is why is there nothing connected to col2 pin? That is confusing.
3. Mapping of col0, col1, col2 and row0 pins to atmega32u4 pins. You mentioned somewhere this mapping (on reddit and geekhack) but it is not present here, and it is needed.
4. Flashing, I got stuck here for some time. I was doing it from Archlinux and since you mentioned that it should be similar to flashing the ProMicro and we can use avrdude I went in that direction but lost a looot of time trying to make that work, and I was not able to -> I could not get Postage Board to appear as /dev/tty* which I needed for avrdude. Why I thought that needs to happen at all? Because I was following guide for ProMicro, and that is how ProMicro works, and since it was mentioned that Postage Board works same as ProMicro + avrdude, I thought that could be it. This is of course on me because I don't know much about flashing or mcu's in general, but on the other hand, I think giving some basic information could help others a lot. For example, I just learned following on Reddit: `The Postage Board uses the DFU bootloader instead of the Caterina bootloader that the Pro Micros use, so it doesn't show up as a TTY device.` -> so this is super useful, it would save 4 hours of debugging for me. Can it even work with avrdude then? I don't know because I don't have enough experience there, but if it can, it would be great to see an example of this. What I know it works with though is dfu-programmer, so that is worth mentioning. I learned this by running `make <kbname>:<keymap>:dfu` in QMK which just worked, and then I dig in to figure out it uses dfu-programmer, which works because `It works because :dfu uses dfu-programmer, not dfu-util - the latter is for boards with STM chips and the former is for your atmega32u4's DFU bootloader.` -> as another reddit user said.
To summarize this part, I think just mentioning that it is different from ProMicro because it uses DFU and not Caterina bootloader and that it works with dfu-programmer, does not appear as tty and can be easily flashed with `:dfu` from qmk, that would already be great info.

I can make a PR if needed although I believe you will be able to shape this much better, since you know much more than me.
Thanks!

**Comment by davidnite** [comment]
Documenting the answer to 3 here, since I had to go searching for this as well:

Col0 is on B7
Col1 is on C6
Col2 is on C7
Row0 is B6
